pages/index.js

```diff
@@ -1,109 +1,78 @@
-import { useContext } from "react";
 import Row from "react-bootstrap/Row";
 import Col from "react-bootstrap/Col";
-import Image from "react-bootstrap/Image";
-import { FaGlobe, FaUsersCog, FaBusinessTime } from "react-icons/fa";
+import { IoIosPeople } from "react-icons/io";
+import { FaGlobe, FaBriefcase } from "react-icons/fa";
 import { MdWeb } from "react-icons/md";
-import IssuesContext from "../contexts/IssuesContext";
 import Page from "../components/shared/Page";
-import IndexIsssueListItem from "../components/shared/IndexIssueListItem";
+import FeaturedCarousel from "../components/FeaturedCarousel";
 
 function ServiceItem(props) {
   const { icon, content } = props;
   return (
-    <Col md={3} className="pb-4">
+    <Col className="px-3">
       <Row>
-        <Col xs="auto" className="pe-0">
-          <span style={{ color: "#00bfa5" }}>
-            <h1>{icon}</h1>
-          </span>
-        </Col>
-        <Col>
-          <h5>{content}</h5>
-        </Col>
+        <h1>{icon}</h1>
+      </Row>
+      <Row>
+        <p style={{ fontSize: "1.1rem" }}>{content}</p>
       </Row>
     </Col>
   );
 }
 
 export default function HomeView() {
-  const { projectIssues: projects } = useContext(IssuesContext);
-
   return (
     <>
       <Page fluid nav hideSeparator>
-        <Row>
-          <Col xs={1} md={2}></Col>
-          <Col sm={12} md={9} lg={4} className="py-5 px-4 text-primary">
-            <h1 className="homepage-header">
-              <strong>Tech support for ATX mobility</strong>
+        <Row className="hero-section d-flex align-items-center">
+          <Col>
+            <h1 className="text-white display-3 fw-bold text-center pb-4">
+              Tech Support for ATX Mobility
```

Comment:
Short words are good words in a hero image, but I get the heebie jeebies saying "tech support." is it just me? IT Support? Information Technology Support? Technology Support?